playbooks/reboot_cluster.yml

```diff
@@ -78,26 +78,43 @@
     until: result | succeeded
     retries: 10 
 
-- hosts: ceph_osds[0]
+- hosts: ceph_osds
+  serial: 1
   tasks:
+  - name: ceph health check before starting reboots
+    command: /etc/sensu/plugins/check-ceph.rb
+    register: result
+    until: result | succeeded
+    retries: 10
+    run_once: true
```

Comment:
Serial 1 and run_once one are an interesting mix. run_once I believe is _per_ _batch_ which means with a serial of 1, it'll run for every host.

What is the intent here? Any reason why we can't just run it every host, even if it means running 2x in rapid succession (end of play, start of play)?